### ISSUE TITLE
Trying to run .\run-demo.ps1 from Master

### ISSUE THREAD

**Issue by TFAstudio** [aaronpowell/Chauffeur#75]
I am interested in seeing whether we can incorporate this, but when I clone the repository, and the run .\run-demo.ps1, I get errors!  Any help would be appreciated:

```
Building project with version: LocalBuild
Shortened DependencyGraph for Target Build:
<== Build
   <== AssemblyInfo
   <== Clean
   <== RestoreChauffeurTestsPackages
      <== RestoreChauffeurDemoPackages
         <== RestoreChauffeurTestingToolsPackages
            <== RestoreChauffeurPackages

The resulting target order is:
 - AssemblyInfo
 - Clean
 - RestoreChauffeurPackages
 - RestoreChauffeurTestingToolsPackages
 - RestoreChauffeurDemoPackages
 - RestoreChauffeurTestsPackages
 - Build
Starting Target: AssemblyInfo
  NoDescription
git.exe log --oneline -1
Finished Target: AssemblyInfo
Running build failed.
Error:
System.Exception: git log --oneline failed: fatal: not a git repository (or any of the parent directories): .git
   at Microsoft.FSharp.Core.PrintfModule.PrintFormatToStringThenFail@1379.Invoke(String message)
   at Fake.Tools.Git.Information.shortlog(String repositoryDir) in C:\proj\FAKE\src\app\Fake.Tools.Git\Information.fs:line 94
   at Fake.Tools.Git.Information.getCurrentHash() in C:\proj\FAKE\src\app\Fake.Tools.Git\Information.fs:line 110
   at FSI_0005.Build.clo@66-1.Invoke(TargetParameter _arg1)
   at Fake.Core.Target.runSingleTarget(Target target) in C:\proj\FAKE\src\app\Fake.Core.Target\Target.fs:line 449

---------------------------------------------------------------------
Build Time Report
---------------------------------------------------------------------
No target was successfully completed
---------------------------------------------------------------------
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.AggregateException: Target 'AssemblyInfo' failed. ---> System.Exception: git log --oneline failed: fatal: not a git repository (or any of the parent directories): .git
   at Microsoft.FSharp.Core.PrintfModule.PrintFormatToStringThenFail@1379.Invoke(String message)
   at Fake.Tools.Git.Information.shortlog(String repositoryDir) in C:\proj\FAKE\src\app\Fake.Tools.Git\Information.fs:line 94
   at Fake.Tools.Git.Information.getCurrentHash() in C:\proj\FAKE\src\app\Fake.Tools.Git\Information.fs:line 110
   at FSI_0005.Build.clo@66-1.Invoke(TargetParameter _arg1)
   at Fake.Core.Target.runSingleTarget(Target target) in C:\proj\FAKE\src\app\Fake.Core.Target\Target.fs:line 449
   --- End of inner exception stack trace ---
   at Fake.Core.Target.run$cont@476(String targetName, Unit unitVar) in C:\proj\FAKE\src\app\Fake.Core.Target\Target.fs:line 543
   at Fake.Core.Target.run(String targetName) in C:\proj\FAKE\src\app\Fake.Core.Target\Target.fs:line 476
   at Fake.Core.Target.RunOrDefault(String defaultTarget) in C:\proj\FAKE\src\app\Fake.Core.Target\Target.fs:line 571
   at <StartupCode$FSI_0005>.$FSI_0005_Build$fsx.main@()
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor)
   at System.Reflection.RuntimeMethodInfo.UnsafeInvokeInternal(Object obj, Object[] parameters, Object[] arguments)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at Fake.FSIHelper.runScriptCached@329.Invoke(Unit unitVar0) in C:\proj\FAKE\src\app\FakeLib\FSIHelper.fs:line 337
System.AggregateException: Target 'AssemblyInfo' failed. ---> System.Exception: git log --oneline failed: fatal: not a git repository (or any of the parent directories): .git
   at Microsoft.FSharp.Core.PrintfModule.PrintFormatToStringThenFail@1379.Invoke(String message)
   at Fake.Tools.Git.Information.shortlog(String repositoryDir) in C:\proj\FAKE\src\app\Fake.Tools.Git\Information.fs:line 94
   at Fake.Tools.Git.Information.getCurrentHash() in C:\proj\FAKE\src\app\Fake.Tools.Git\Information.fs:line 110
   at FSI_0005.Build.clo@66-1.Invoke(TargetParameter _arg1) in D:\Projects\Test\Chauffeur-master\build.fsx:line 67
   at Fake.Core.Target.runSingleTarget(Target target) in C:\proj\FAKE\src\app\Fake.Core.Target\Target.fs:line 449
   --- End of inner exception stack trace ---
   at Fake.Core.Target.run$cont@476(String targetName, Unit unitVar) in C:\proj\FAKE\src\app\Fake.Core.Target\Target.fs:line 543
   at Fake.Core.Target.run(String targetName) in C:\proj\FAKE\src\app\Fake.Core.Target\Target.fs:line 476
   at Fake.Core.Target.RunOrDefault(String defaultTarget) in C:\proj\FAKE\src\app\Fake.Core.Target\Target.fs:line 571
   at <StartupCode$FSI_0005>.$FSI_0005_Build$fsx.main@() in D:\Projects\Test\Chauffeur-master\build.fsx:line 258
---> (Inner Exception #0) System.Exception: git log --oneline failed: fatal: not a git repository (or any of the parent directories): .git
   at Microsoft.FSharp.Core.PrintfModule.PrintFormatToStringThenFail@1379.Invoke(String message)
   at Fake.Tools.Git.Information.shortlog(String repositoryDir) in C:\proj\FAKE\src\app\Fake.Tools.Git\Information.fs:line 94
   at Fake.Tools.Git.Information.getCurrentHash() in C:\proj\FAKE\src\app\Fake.Tools.Git\Information.fs:line 110
   at FSI_0005.Build.clo@66-1.Invoke(TargetParameter _arg1) in D:\Projects\Test\Chauffeur-master\build.fsx:line 67
   at Fake.Core.Target.runSingleTarget(Target target) in C:\proj\FAKE\src\app\Fake.Core.Target\Target.fs:line 449<---
Stopped due to error

Push-Location : Cannot find path 'D:\Projects\Test\Chauffeur-master\Chauffeur.Demo\bin' because it does not exist.
At D:\Projects\Test\Chauffeur-master\run-demo.ps1:4 char:1
+ Push-Location .\Chauffeur.Demo\bin
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (D:\Projects\Tes...uffeur.Demo\bin:String) [Push-Location], ItemNotFound
   Exception
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.PushLocationCommand

./chauffeur.runner.exe : The term './chauffeur.runner.exe' is not recognized as the name of a cmdlet, function, script
file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct
and try again.
At D:\Projects\Test\Chauffeur-master\run-demo.ps1:5 char:1
+ ./chauffeur.runner.exe delivery "-p:pwd=password!1"
+ ~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (./chauffeur.runner.exe:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

Copied template config file 'C:\Program Files (x86)\IIS Express\AppServer\applicationhost.config' to 'C:\Users\adelboy\AppData\Local\Temp\iisexpress\applicationhost2018813110367.config'
Updated configuration file 'C:\Users\adelboy\AppData\Local\Temp\iisexpress\applicationhost2018813110367.config' with given cmd line info.
Starting IIS Express ...
Failed to register URL "http://localhost:8080/" for site "Development Web Site" application "/". Error description: Cannot create a file when that file already exists. (0x800700b7)
Registration completed
Unable to start iisexpress.

Cannot create a file when that file already exists.
For more information about the error, run iisexpress.exe with the tracing switch enabled (/trace:error).
```

**Comment by aaronpowell** [comment]
Whoops, the `run-demo.ps1` script didn't build the demo site. This is fixed with aa59f69.

Regarding the `git` error, was it in a git repo that you ran it from? The `FAKE` script expects it to be a git repo so it can generate some stuff.

**Comment by TFAstudio** [comment]
Yes - that is fine - with a git pull that error disappears.  Thanks for your help!
